marti_nav_msgs/msg/Plan.msg

```diff
@@ -0,0 +1,13 @@
+# This message represents a plan along a roadway
+std_msgs/Header                header
+
+# List of points in the plan
+PlanPoint[]                    points
+
+# Location for placing custom per point properties
+# Used for unusual things such as intersections or crosswalks
+PlanPointProperty[]            point_properties
+
+# Per route details
+string id
```

Comment:
`id` should be renamed to `plan_id` to be consistent.

Reply:
I disagree that this would be more consistent and I think it is a smell to have the name of the message used in the fields of it. The name below is plan_id to indicate it is the id of the plan so it contains additional information.

Reply:
also `id` is a built in function in Python, so anyone using these messages in Python would be creating ambiguity. It doesn't need to be `plan_id`, but I would rather not call anything `id`